Relativity.Processing.SDK.md

```diff
@@ -4,20 +4,41 @@
 
 This package contains interfaces for the public APIs of the Relativity Processing application.
 
-## 11.3.101.10
+## 12.2.13
 
 ### Release Notes
 
-* Initial version
+* Updated DFV to properly populate Processing Deletion fields.
+* Ported Post Publish Delete performance improvement changes from mayapple.
+* Bumped the Invariant Services package for ARM statistics endpoint defect fix. Fixed error field removals.
+* Fix adding index to zprocessingdochistory table.
+* Advanced Picker pagination support in files tab.
+* Removing toggle for Processing Profile Inclusion/Exclusion feature.
+* ARM Processing: Do not throw exception for duplicate cancel requests for canceled or canceling workspaces and Fix Processing Exists readiness check defect.
+* Replaced remaining WCF endpoints with Kepler endpoints.
+* Added a NuGet package for Password Bank, renamed Password Bank tab, and moved from ApplicationBase.
+* Added Dedupe Status Field to Files tab default view.
+* Resolved dependency injection issues and removed low-use RSAPI code, along with legacy test projects.
+* Removed redundant Kepler endpoint check for ValidateApiEndpoints API, removed ProcessingBase package and consolidated projects into Relativity.Eca.
+* Added a new private endpoint for updating count after Post Publish Delete job.
+* Added APM to PreMassDeleteEH.
+* Enabled OI PowerPoint toggle.
+* Moving Processing Set page interactions to web assets from MVC.
+* Use Object Manager instead of RSAPI for Custodian, Processing Set and Processing Data Source updates.
+* Updates for Error Workflow, unhide Document Errors tab.

```

Comment:
Maybe rephrase this, I don't think "unhide Document Errors tab" applies to 11.3.101.10 -> 12.2.13